pkg/serverutils/knative_event_source_utils.go

```diff
@@ -0,0 +1,55 @@
+package serverutils
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// KnativeEventSourceSpec describes how a user wants their resource to appear
+type KnativeEventSourceSpec struct {
```

Comment:
So I dont thingthis is the right place where these types should live. openshift/api repo should be the proper place. @spadgett thoughts ?

Reply:
Since its a temporary workaround I think its OK, to put it here.